platform/ios/PolyPodApp/PolyPod/Authentication.swift

```diff
@@ -61,12 +88,12 @@ class Authentication {
         )
     }
     
-    private func isSetUp() -> Bool {
+    func isSetUp() -> Bool {
         return UserDefaults.standard.bool(forKey: Authentication.setUpKey)
     }
     
     private func authenticateLocally(
-        reason: String,
+        _ reason: String,
```

Comment:
Any reason why you change this?

Reply:
there's a use of this function that the passed string was a var already...and just didn't like the double `reason: reason` - just matter of taste. I can revert it. (Just prefer the way TS does it tbh where it doesn't need to explicitly set the label when the var has the same name :/)

Reply:
Right, it does not make a lot of sense. Didn't seem related, though.

Reply:
Consider this:
```
func authenticateLocally(withReason reason: String, _ completionAction: @escaping (Bool) -> Void)
```
then the function will be called like this:
```
authenticateLocally(withReason: reason) { ...}
```